content/docs/querying/operators.md

```diff
@@ -195,7 +196,7 @@ If the metric `http_requests_total` had time series that fan out by
 `application`, `instance`, and `group` labels, we could calculate the total
 number of seen HTTP requests per application and group over all instances via:
 
-    sum(http_requests_total) by (application, group)
+    sum(http_requests_total) without (instance)
```

Comment:
How about adding that this is equivalent to `sum(http_requests_total) by (application, group)`?

Actually, would you say `without` is always preferable? I'm just thinking of situations where one wants to keep output labels constant even when additional labels are added to a metric. And also, `by` lists more explicitly what labels are actually still there after the aggregation. With `without`, I have to know what labels exist in the first place to know what labels an expression outputs when looking at it.